Sources/Extensions/Widgets/Sensor/WidgetSensorsView.swift

```diff
@@ -0,0 +1,53 @@
+import Shared
+import SwiftUI
+import WidgetKit
+
+@available(iOS 17, *)
+struct WidgetSensorsView: View {
+    @Environment(\.widgetFamily) var family: WidgetFamily
+
+    var entry: WidgetSensorsEntry
+
+    private static let maxItems: Int = 3
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            if entry.sensorData.count != WidgetSensorsView.maxItems {
+                Image(uiImage: Self.scaleLogo(logo: Asset.SharedAssets.logo.image, size: 30))
```

Comment:
Instead of `UIGraphicsImageRenderer` can you use `.resizable` modifier and then set the frame for it? This is way less intensive operation, widget extension is very sensetive for memory and processing power usage